bump.go

```diff
@@ -14,6 +14,8 @@ var (
 	errNoUpdate      = errors.New("no resources to update")
 	errSkipBadCommit = errors.New("skipping bump, as the latest commit is already by the bumper tool")
 	errFailedMeta    = errors.New("failed to open plasma.yaml")
+	errLastCommit    = errors.New("last commit")
```

Comment:
can you please expand error message for both variables ?

Reply:
removed